scripts/prepare_network.py

```diff
@@ -225,18 +225,18 @@ def set_transmission_limit(n, ll_type, factor, costs, Nyears=1):
 
 
 def average_every_nhours(n, offset):
-    logger.info(f"Resampling the network to {offset}")
+    logger.info(f"Resampling the network to {offset.casefold()}")
     m = n.copy(with_time=False)
 
-    snapshot_weightings = n.snapshot_weightings.resample(offset).sum()
+    snapshot_weightings = n.snapshot_weightings.resample(offset.casefold()).sum()
```

Comment:
My feeling is that it's a perfectly appropriate solution, as keeping the user interfaces must have a priority over code elegance. I'm only thinking that probably it could make sense to add a comment to this line explaining that the case transformation is needed to keep `H` in the options definitions. What do you think @finozzifa?

Reply:
I agree. Done!